reports/Oregon.md

```diff
@@ -1,5 +1,41 @@
 ## Eugene
 
+### Peaceful protestor tear gassed | May 30th
+
+Footage shows a small cluster of protestors chanting "I can't breathe." At 1:15, without warning, police fire tear gas. Most protestors flee. Some canisters can be seen thrown back towards police.
```

Comment:
Maybe make the second sentence  "Footage shows a small cluster of protestors chanting "I can't breathe." At 1:15, without warning, police fire tear gas causing most protestors to flee."